### ISSUE TITLE
Build cache

### ISSUE THREAD

**Issue by kettanaito** [kettanaito/remix-og-image#19]
There are two outcomes of this task:

1. Research how to enable and use build caching to skip image generation.
2. Document how to set up caching in README.

## Cloudflare

- Use [Build Caching](https://developers.cloudflare.com/pages/configuration/build-caching) (has to be enabled explicitly). Supports some major frameworks but not Remix. 

If Remix was supported, enabling Build Caching would theoretically be enough to cache the generated images, but now you get this:

```
Skipping build output cache as it's not supported for your project
```

## Vercel

- Use [Build caching](https://vercel.com/docs/deployments/troubleshoot-a-build#what-is-cached). I don't seen an explicit mention of Remix build cache path, but Vercel caches everything in `node_modules` across builds. The plugin can utilize that.

**Comment by kettanaito** [comment]
Right now, the local cache is broken because the plugin checks if an image exist at the cached path, and it will never exist since Vite clears the `build/client` directory before each build. 

## How to solve this

The plugin should output the images to the cache directory so if there's a cache match, it will simply copy them from cache to `build/client` without having to spawn browser tabs.

**Comment by kettanaito** [comment]
## Released: v0.3.5 🎉

This has been released in v0.3.5!

- 📄 [**Release notes**](https://github.com/kettanaito/remix-og-image/releases/tag/v0.3.5)
- 📦 [npm package](https://www.npmjs.com/package/remix-og-image/v/0.3.5)

Make sure to always update to the latest version (`npm i remix-og-image@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.
